themes/theme-ios11/widgets/selectors.js

```diff
@@ -92,13 +92,50 @@ const getResultByHash = createSelector(
  * @param {Object} params The query parameters.
  * @returns {Object} The product result.
  */
-export const getProductsResult = createSelector(
+export const getProductsResultWithHash = createSelector(
   state => state,
   getResultHash,
   getResultByHash,
   getPopulatedProductsResult
 );
 
+/**
+ * @param {Object} state he application state.
+ * @param {int} type Number indicating search type.
+ * @param {Object} params Query parameters.
+ * @param {string} id Id of component.
+ * @return {Object} The product result.
+ */
+export const getProductsResultsWithIds = (state, type, params, id) => {
```

Comment:
Some products will be returned here, even if the action was not yet called from the widget. 

So, if some products, defined in the config, are already in redux, the missing once would never be called.
(check, widgets/Products/index.jsx:componentDidMount).

Reply:
I think the componentDidMount should be extended to not check for the current condition if widget is setup for product numbers.

Reply:
done